frontend/src/screens/HomeScreen.tsx

```diff
@@ -1,13 +1,23 @@
+import * as React from 'react';
 import { Container } from '@mui/material';
 import Map from 'components/Map';
-import React from 'react';
+import BigPopup from 'components/big popup/index';
```

Comment:
Why you call component "big popup"? Using spaces in names is bad practise. Call it BigPopup instead